README.md

```diff
@@ -15,3 +15,20 @@ Removed some code such as:
 * Useless controllers
 * Support for i18n
 * activator
+
+## Getting Started
+
+To run this demo using activator:
+
+ * `git clone` this repository
+ * Update the MySQL server url, username and password in `conf/application.conf`
+ * Create a `playScalaSlickExample` database on your MySQL server.
+
+```mysql
+    CREATE DATABASE playScalaSlickExample;
+```
+
+ * Launch the demo using `activator ~run`
```

Comment:
some starters may not be aware of what is activator, `sbt run` would simplify in that case, no?